### PR TITLE
Fix ie

### PR DESCRIPTION
**Pull request by RobertHebel**
Fix: crash on ie11

### CODE REVIEW

File: docs-www/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "lint:autofix": "eslint --ext .jsx,.js . --fix"
   },
   "dependencies": {
-    "@brainhubeu/gatsby-docs-kit": "https://github.com/RobertHebel/gatsby-docs-kit.git#upgrade-gatsby-to-v2",
+    "@brainhubeu/gatsby-docs-kit": "https://github.com/brainhubeu/gatsby-docs-kit.git#fix-ie",
```
Comment (piotr-s-brainhub) [inline docs-www/package.json]: 👍

Comment (piotr-s-brainhub) [reply docs-www/package.json]: later we have to use the main version

File: plugins.js [hunk 2]
```diff
@@ -87,6 +87,12 @@ function pluginConfigFactory(config) {
         },
       },
     },
+    {
+      resolve: `gatsby-plugin-compile-es6-packages`,
+      options: {
+        modules: ['acorn-jsx', 'regexpu-core', 'unicode-match-property-ecmascript', 'unicode-match-property-value-ecmascript'],
```
Comment (piotr-s-brainhub) [inline plugins.js]: how did you get this list?

Comment (RobertHebel) [reply plugins.js]: I just checked which packages weren't transformed automatically by the webpack and I added them here

**Pull request merged**